le_utils/constants/embed_topics_request.py

```diff
@@ -16,30 +16,50 @@
     "description": "Schema for embed topics requests received by RayServe",
     "additionalProperties": False,
     "definitions": {
-        "ancestors": {
-            "type": "array",
-            "description": "The ancestors of the topic, in order, from the parent to the root",
-            "items": {"$ref": "#/definitions/topic"},
+        "id": {
+            "type": "string",
+            "description": "The ID of the topic content node on Studio",
+        },
+        "title": {"type": "string", "description": "The title of the topic"},
+        "description": {
+            "type": "string",
+            "description": "The description of the topic",
         },
         "language": {
             "type": "string",
             "description": "Language code from https://github.com/learningequality/le-utils/blob/main/le_utils/resources/languagelookup.json",
             "pattern": "^[a-z]{2,3}(?:-[a-zA-Z]+)?$",
         },
+        "level": {
+            "type": "integer",
+            "description": "The level of the ancestor, where the parent is 1 and the root is the highest level",
```

Comment:
When you proposed this, I assumed you would make root the lowest `level`. Having it be the highest means the `level`s always depend on the folder depth at which you're looking up recommendations, meaning you have to recompute the levels on the ancestors in order to send it. 

Django MPTT, which we use on Studio for the hierarchical structure of content nodes, adds a `level` field to the models. In that situation, `level=0` is the root and so on increases as you go deeper in the hierarchy. This `level` being in the inverse of that is likely to cause confusion with those who are familiar with that.

All that said, the root node of a studio channel wouldn't be needed, since the root node of a channel is essentially the channel itself, and we don't use the channel title in the topic representation. So if you make `level` start at 0, saying 0 is the root node gives the impression it's the root channel node. So if that's the case, if a `level=0` was set, it could be ignored, otherwise it always uses level >=1

Overall, reversing the level would better align with our ecosystem's content channel architecture.

If you have strong feelings, an alternative would be to add the `parent` field to each. Then it's very clear which node is the root, because it's the node without a parent. But I don't think that's necessary and it makes sorting the ancestors a little more complicated than a simple `level` does.

Reply:
Ahh okay, I see. I think this change shouldn't be much of a lift on either side(le-utils and curriculum-automation). I didn't think about the order too much, but I agree this is confusing for one coming in with knowledg on studio's hierarchical structure.